src/ChipInput.js

```diff
@@ -343,6 +343,16 @@ class ChipInput extends React.Component {
         }
       }
 
+      if (typeof chip === 'object') {
```

Comment:
Would you mind changing this to `else if (typeof chip === 'object')`?

Reply:
Done

Reply:
Did you push your changes? I don't see it yet.

Reply:
Wanted to push everything together